deploy/operator/setup_lso.sh

```diff
@@ -0,0 +1,61 @@
+source utils.sh
+
+set -xeo pipefail
+
+function install_lso() {
+    oc adm new-project openshift-local-storage || true
+
+    oc annotate project openshift-local-storage openshift.io/node-selector='' --overwrite=true
+
+    cat <<EOCR | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: local-operator-group
+  namespace: openshift-local-storage
+spec:
+  targetNamespaces:
+  - openshift-local-storage
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: local-storage-operator
+  namespace: openshift-local-storage
+spec:
+  installPlanApproval: Automatic
+  name: local-storage-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOCR
+
+    wait_for_operator "local-storage-operator" "openshift-local-storage"
+}
+
+function create_local_volume() {
+    wait_for_crd "localvolumes.local.storage.openshift.io" "openshift-local-storage"
+
+    cat <<EOCR | oc apply -f -
+apiVersion: local.storage.openshift.io/v1
+kind: LocalVolume
+metadata:
+  name: assisted-service
+  namespace: openshift-local-storage
+spec:
+  logLevel: Normal
+  managementState: Managed
+  storageClassDevices:
+    - devicePaths:
+        - /dev/sdb
+        - /dev/sdc
+        - /dev/sdd
+        - /dev/sde
+        - /dev/sdf
```

Comment:
I would suggest for this list to be configurable rather than hard-coding the paths. Different use-cases (dev-scripts, prow, local testing) may use different paths.

Reply:
sure thing. Will be handled on MGMT-5419